### PR TITLE
configure consumer sets

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: croud/consumersets/commands.py [hunk 1]
```diff
@@ -50,12 +47,46 @@ def consumer_sets_list(args: Namespace) -> None:
...
 
     query = Query(body, args, endpoint="/product/graphql")
     query.execute(vars)
     print_query(query, "allConsumerSets")
+
+
+def consumer_sets_edit(args: Namespace) -> None:
+    body = dedent(
+        """
+    mutation ConfigureConsumerSet(id: String!, input: ConfigureConsumerSetInput!) {
```
Comment (m-kharbat) [inline croud/consumersets/commands.py]: just out of curiosity, does this work ? isn't it required to have `$` in variable names: 

`mutation ConfigureConsumerSet(id: String!, input: ConfigureConsumerSetInput!)` > `mutation ConfigureConsumerSet($id: String!, $input: ConfigureConsumerSetInput!)`

Comment (alijared) [reply croud/consumersets/commands.py]: @m-kharbat is right

File: croud/consumersets/commands.py [hunk 2]
```diff
@@ -50,12 +47,46 @@ def consumer_sets_list(args: Namespace) -> None:
     """  # noqa
     ).strip()
 
-    vars = {
+    vars = clean_dict({
```
Comment (MarkusH) [inline croud/consumersets/commands.py]: No need here. The server will treat empty and NULL as not provided.

Comment (chaudum) [reply croud/consumersets/commands.py]: I agree, it's not necessary, but it's safer. And it would be needed if arguments have default values.

**Pull request merged**